docs/admin/install.rst

```diff
@@ -0,0 +1,265 @@
+Installing and Configuring SecureDrop Workstation
+=================================================
+
+.. include:: ../includes/top-warning.rst
+
+Prerequisites
+-------------
+In order to install SecureDrop Workstation and configure it to use an existing SecureDrop instance, you will need the following:
+
+- A Qubes-compatible computer with at least 16GB of RAM (32 GB is recommended). SecureDrop Workstation has mainly been tested against Lenovo 6th-gen T470 and X1 models, but there are no formally supported hardware recommendations at this time. See Qubes' `Hardware Compatibility List <https://www.qubes-os.org/hcl/>`_ for more options.
```

Comment:
We've in fact recommended the T480 for all pilot participants; if we want all folks who install SecureDrop Workstation to update the BIOS, then I think we need to be a bit more explicit about how to do that; the support site is not helpful. Should folks keep a pre-existing Windows partition in place to perform an update? Should they use the bootable CD or the Linux binary? The more we can take the guesswork out of it for the primary hardware platform we're installing on, the better, IMO.

Reply:
Steps that worked for me on T480 for BIOS update:
1. Download .iso image from https://pcsupport.lenovo.com/us/en/products/laptops-and-netbooks/thinkpad-t-series-laptops/thinkpad-t480-type-20l5-20l6/downloads/ds502355
2. Use `geteltorito` to create bootable image
3. Use `dd` to create bootable USB drive
4. Boot from USB and follow instructions.

Reply:
BIOS updates have been split out to a separate hardware page in latest commit.